### DISCUSSION TITLE
key_mgmt

### DISCUSSION THREAD

**Discussion by dav322** [WLAN-Pi/feedback#56] [General Inquiries]
I was having a discussion with chatgpt about the necessity of providing the key_mgmt value.  It was under the impression that you had to specify a key_mgmt if it was needed for connection.

Windows only asks for a ssid name and passphrase and connects to all the Wi-Fi networks.

Why do you need anything else but ssid name and pass phrase to connect a raspberry pi to a Wi-Fi network?  Providing anything but ssid and passphrase is beyond the capabilities for 99.9% of computer users.

I am using the pi without keyboard and mouse from communicating from windows computers.  How would I even tell a customer what value they should put in the key_mgmt parameter?

**Comment by joshschmelzle** [comment]
Refer to https://w1.fi/cgit/hostap/plain/wpa_supplicant/wpa_supplicant.conf for the various key_mgmt configuration possibilities.
